boards/st/nucleo_h745zi_q/nucleo_h745zi_q_stm32h745xx_m7.dts

```diff
@@ -144,3 +144,27 @@ zephyr_udc0: &usbotg_fs {
 	pinctrl-names = "default";
 	status = "okay";
 };
+
+&usbotg_fs {
+	pinctrl-0 = <&usb_otg_fs_sof_pa8
+		     &usb_otg_fs_vbus_pa9
+		     &usb_otg_fs_dm_pa11
+		     &usb_otg_fs_dp_pa12>;
+	pinctrl-names = "default";
+	status = "disabled";
```

Comment:
Nodes added in boards should all be enabled.

Reply:
fdcan1 conflicts with i2c1 pinouts. Moving fdcan1 to another subset of pins will not work with waveshark rs485-can hat matching perfectly on the board. Thus I have to disable fdcan1 since i2c1 was as first one there. I've also added a comment, what needed to be done to enable fdcan1.

Reply:
fdcan1 will become pins mentioned by @FRASTM